frontend/packages/console-shared/src/utils/icon-utils.ts

```diff
@@ -0,0 +1,11 @@
+import * as _ from 'lodash';
+import {
+  ClusterServiceVersionKind,
+  ClusterServiceVersionIcon,
+} from '@console/operator-lifecycle-manager';
+import * as operatorLogo from '../images/operator.svg';
+
+export const getImageForCSVIcon = (csv: ClusterServiceVersionKind) => {
+  const icon: ClusterServiceVersionIcon = _.get(csv, 'spec.icon', []);
+  return !_.isEmpty(icon) ? `data:${icon[0].mediatype};base64,${icon[0].base64data}` : operatorLogo;
```

Comment:
Importing `'@console/operator-lifecycle-manager'` from `@console-shared` is bad.
To remove this dependency, perhaps you type the param as much simpler type to match the CSV icon:
```
type Icon = { base64data: string; mediatype: string };

export const getOperatorIcon = (icon: Icon[]) => {
```